firmware/chipcon_usb.c

```diff
@@ -1168,6 +1170,11 @@ void processOUTEP5(void)
                 appReturn(2, ptr);
                 break;
 
+            case CMD_DEVICE_SERIAL_NUMBER:
+		convertUsbDeviceSerialNumberToString(deviceSerialNumber);
```

Comment:
this makes sense for a test setup, but i can't mainline this.
this should return the bytes from the bootloader serial number, not processed.  do the conversion in python.  it's easier there anyway.